### PR TITLE
Added "navigate to" entry and navigating through multiple GUI options

### PR DESCRIPTION
**Pull request by sffrench**
Added "navigate to" entry and "navigating through multiple GUI options"

Addresses #188

### CODE REVIEW

File: en-US/Design.xml [hunk 1]
```diff
@@ -91,7 +91,16 @@
 			 <para>
 				See the <citetitle>Computer Interfaces</citetitle> chapter in <citetitle>The IBM Style Guide</citetitle> for more information.
 			</para>
+			<section>
+			 <title>Navigating through multiple GUI options</title>
```
Comment (daobrien) [inline en-US/Design.xml:95]: Remember to use title case for titles.

File: en-US/N.xml [hunk 2]
```diff
@@ -6,6 +6,15 @@
 <chapter id="n">
 	<title>N</title>
 	 <variablelist>
+		 <varlistentry id="navigate-to">
+ 			<term>navigate to</term>
+ 			 <listitem>
+ 				<para>
+					Use "Navigate to" when moving through multiple GUI options because it covers all cases where you might have to click, point to, select, or otherwise make a series of selections to initiate an action. For example, "From the OpenShift web console, navigate to Monitoring → Alerting."
```
Comment (daobrien) [inline en-US/N.xml:13]: I would add a second para here that says something like "Do not use 'Go to' or 'point to' or other variations."

File: en-US/Design.xml (hunk 1 above)
Comment (sffrench) [reply en-US/Design.xml]: Thanks, David, that's done.

File: en-US/N.xml (hunk 2 above)
Comment (sffrench) [reply en-US/N.xml:13]: Good advice David, that's done.

**Review by daobrien — APPROVED** [review]
Sorry this took so long. All good.

**Pull request merged**